api/v1alpha3/webhook_suite_test.go

```diff
@@ -52,11 +51,9 @@ func TestAPIs(t *testing.T) {
 }
 
 func TestMain(m *testing.M) {
-	code := 0
-	defer func() { os.Exit(code) }()
```

Comment:
This one will consume any panic, can confirm by adding a panic in any of the tests in the package.

```shell
~/Code/cluster-api-provider-aws(release-0.7)$ git diff
diff --git bootstrap/eks/api/v1alpha3/conversion_test.go bootstrap/eks/api/v1alpha3/conversion_test.go
index 9a3025df..b5b0449c 100644
--- bootstrap/eks/api/v1alpha3/conversion_test.go
+++ bootstrap/eks/api/v1alpha3/conversion_test.go
@@ -27,6 +27,7 @@ import (
 )
 
 func TestFuzzyConversion(t *testing.T) {
+       panic("test")
        g := NewWithT(t)
        scheme := runtime.NewScheme()
        g.Expect(AddToScheme(scheme)).To(Succeed())
~/Code/cluster-api-provider-aws(release-0.7)$ go test -timeout 30s sigs.k8s.io/cluster-api-provider-aws/api/v1alpha3
ok  	sigs.k8s.io/cluster-api-provider-aws/api/v1alpha3	0.021s
```

Reply:
same for all other cases where we have `defer func() { os.Exit(code) }()` or similar.